operator/pkg/ciliumendpointbatch/manager.go

```diff
@@ -43,6 +43,10 @@ type cebTracker struct {
 	ceb *cilium_v2.CiliumEndpointBatch
 	// set of CEPs to be removed in the CEB object in next sync.
 	removedCeps map[string]struct{}
+	// number of CEPs inserted in a CEB
+	cepInserted int64
+	// number of CEPs removed from a CEB
+	cepRemoved int64
```

Comment:
It would seem more natural to just directly increment metric counter here and it won't need to be reset afterwards.

Reply:
The requirement for this metrics is to collect total number of CEPs Inserts/Removal for every CEB sync with k8s-apiserver. As well, we would like to plot number of CEPs Inserts/Removal  in various buckets.

number of CEBs = for <0-10> CEP Inserts/Removal 
number of CEBs = for <11-20> CEP Inserts/Removal 
number of CEBs = for <21-30> CEP Inserts/Removal 
...
number of CEBs = for <91-100> CEP Inserts/Removal

Reply:
I see, that makes sense :)